HW2/P3/P3.txt

```diff
@@ -0,0 +1,26 @@
+Only Thread Parallelism
+
+1 thread
+1074.656613 Million Complex FMAs in 5.37143611908 seconds, 200.068769166 million Complex FMAs / second
+
+2 threads
+1074.656613 Million Complex FMAs in 2.63553118706 seconds, 407.757122464 million Complex FMAs / second
+
+4 threads
+1074.656613 Million Complex FMAs in 1.49925899506 seconds, 716.791839531 million Complex FMAs / second
+
+As we increase the number of threads, we are able to perform more Complex FMAs / second. 
+
+
+Instruction Parallelism as well
+
+1 thread
+1074.94829 Million Complex FMAs in 1.70161700249 seconds, 631.721643842 million Complex FMAs / second
+
+2 threads
+1069.252519 Million Complex FMAs in 1.37672805786 seconds, 776.662110498 million Complex FMAs / second
```

Comment:
I understand that you may have some more FMAs with AVX, but I am not sure I understand how it is possible to get less FMAs.